roles/olm/files/18-upstreamcomponents.configmap.yaml

```diff
@@ -0,0 +1,462 @@
+##---
+# Source: olm/templates/18-upstreamcomponents.configmap.yaml
+
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: upstream-components
+  namespace: operator-lifecycle-manager
+  labels:
+    tectonic-operators.coreos.com/managed-by: tectonic-x-operator
+
+data:
+  customResourceDefinitions: |-
```

Comment:
I think those are already being installed in `openshift-metering` role?

Reply:
This populates the OLM catalog, even though with metering enabled the CRDs will already be installed. See: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/philosophy.md#implementation

OLM won't recreate the CRDs if they already exist.